app/views/layouts/question.html.erb

```diff
@@ -1,3 +1,10 @@
+<% content_for :page_title do %>
+  <% if @form&.errors.present? %>
```

Comment:
I think `if @form.error_summary.present?` would DRY up the error-visibility logic a little bit, but I don't see an issue with it as written. (it looks like we don't use the `exclude_keys` option anywhere, anyway)